File: Code Chef/JavaScript/600-699/649 Mario and Transformation TRANSFORM.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => Number(strItem.trim()));
    
    return intArray;
};

const checkMarioSize = (mushrooms) => {
    const isNormal = (mushrooms % 3) === 0;
    const isHuge = (mushrooms % 3) === 1;
    
    (isNormal)
        ? console.log('NORMAL')
        : (isHuge)
            ? console.log('HUGE')
            : console.log('SMALL');
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((mushrooms) => {
        checkMarioSize(mushrooms);
    });
});

File: Code Chef/C Programming/600-699/649 Mario and Transformation TRANSFORM.c
#include <stdio.h>
int main() 
{
    int t,x;
    scanf("%d",&t);
    for(int i=0;i<t;i++)
    {
        scanf("%d",&x);
        if((x%3)==0)
        {
            printf("NORMAL\n");
        }
        else if((x%3)==1)
        {
            printf("HUGE\n");
        }
        else
        {
            printf("SMALL\n");
        }
    }
	return 0;
}